.github/workflows/publish-beta-release.yml

```diff
@@ -0,0 +1,74 @@
+name: Publish Beta Release to Github Only
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  save_artifacts:
+    name: 'Build VSIX Files'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.13.x'
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.ref }}
+      - name: Create and Package Extensions
+        run: |
+          npm install -g lerna
+          npm install
+          npm run compile
```

Comment:
Can we break apart the install and compile steps? I'm thinking we'd want the ability to rerun a job if there was an intermittent failure during install or compile https://github.blog/2022-03-16-save-time-partial-re-runs-github-actions/

Reply:
Did a new build to make sure these changes (and others) were good: https://github.com/randi274/salesforcedx-vscode-fork/actions/runs/2958449333

Reply:
🏆